parser_library/src/processing/preprocessors/cics_preprocessor.cpp

```diff
@@ -1004,6 +1011,27 @@ class cics_preprocessor final : public preprocessor
 
     static bool ignore_line(std::string_view s) { return s.empty() || s.front() == '*' || s.substr(0, 2) == ".*"; }
 
+    static bool is_ignored_line(std::string_view line, size_t line_len_chars)
+    {
+        if (ignore_line(line))
```

Comment:
```suggestion
        if (line.empty() || line.front() == '*' || line.substr(0, 2) == ".*")
```
and drop the ignore_line function?